plugins/airbrake/src/components/EntityAirbrakeWidget/EntityAirbrakeWidget.tsx

```diff
@@ -40,55 +41,50 @@ export const EntityAirbrakeWidget = ({ entity }: { entity: Entity }) => {
   const classes = useStyles();
 
   const projectId = useProjectId(entity);
-  const errorApi = useApi<ErrorApi>(errorApiRef);
   const airbrakeApi = useApi(airbrakeApiRef);
 
-  const { loading, value, error } = useAsync(
-    () => airbrakeApi.fetchGroups(projectId),
-    [airbrakeApi, projectId],
-  );
-
-  useEffect(() => {
-    if (error) {
-      errorApi.post(error);
+  const { loading, value, error } = useAsync(() => {
+    if (!projectId) {
+      return Promise.resolve(undefined);
```

Comment:
you actually don't need the promise.resolve - but no matter, this works just as well!

Reply:
Actually you do need it since the function isn't `async` :)

Whether we should make the function async or not is a whole other question, but since I was calling an async function when calling the API, it made sense to just return that and do away with needing an `async` here.

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/5939277/157635172-369ed3e7-4a66-48f9-bf07-9343af4f159e.png">

Reply:
ah quite right, i am so used to typing it out async that i missed that :)